frontend/src/constants/datatypes.ts

```diff
@@ -42,8 +42,20 @@ export type Config = {
     [key: string]: boolean
   }
   staticAssets: {
-    useStaticCsvs: boolean,
-    analysisResultsRootUrl: string
+    appIsStatic: boolean,
+    csv: {
+      useStaticCsvs: boolean,
+      analysisResultsRootUrl: string
+    }
+    representativePoints: {
```

Comment:
can we consolidate into one static-file-url and have a known folder/file structure instead?

Reply:
Done.